addon/components/base/bs-nav/item.js

```diff
@@ -22,6 +24,22 @@ export default Component.extend(ComponentParent, {
   classNameBindings: ['disabled', 'active'],
   tagName: 'li',
   ariaRole: 'presentation',
+  router: service(),
+
+  /**
+   * If set will wrap the item's content in a link. Accepts either a string with the route name, or an array
+   * with the route name and one or multiple models / query params, similar to the positional params of `{{link-to ...}}`
```

Comment:
Maybe docs should also mention query params?

Reply:
It (briefly) does: "with the route name and one or multiple models / query params". Would you expect/suggest something more verbose?

Reply:
Oh, of course. 😧 Sorry for the noice.